docs/api/pet/delete_pet.md

```diff
@@ -5,3 +5,53 @@ nav_order: 5
 ---
 
 # Delete pet
+
+**DELETE** `/pets/{id}`
+
+Delete the [`pet`](index.md) in the service with the specified `id`.
+
+## URL
+
+```shell
+{base_url}/pets/{id}
+```
+
+### Path Parameters
```

Comment:
Since most of the other headings are sentence case, you may want to make this (and similar headings in other files) sentence case as well for consistency.

Reply:
Thanks for catching! Updated throughout.